bots/c/client_lib/src/internal/grpc_bridge.cpp

```
@@ -0,0 +1,539 @@
+	game.config.bomb_reach = cfg.bomb_reach();
+	game.config.bomb_damage_core = cfg.bomb_damage_core();
+	game.config.bomb_damage_unit = cfg.bomb_damage_unit();
+	game.config.bomb_damage_deposit = cfg.bomb_damage_deposit();
+
+	int unit_count = cfg.units_size();
+	game.config.units = (t_unit_config **)malloc(sizeof(t_unit_config *) * (unit_count + 1));
+	for (int i = 0; i < unit_count; i++)
+	{
+		const auto &u = cfg.units(i);
+		t_unit_config *uc = (t_unit_config *)malloc(sizeof(t_unit_config));
+		uc->name = strdup(u.name().c_str());
```

Cppcheck (2.19.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)